etna/transforms/feature_selection/feature_importance.py

```diff
@@ -52,8 +59,14 @@ def __init__(
         Parameters
         ----------
         model:
-            model to make selection, it should have ``feature_importances_`` property
-            (e.g. all tree-based regressors in sklearn)
+            Model to make selection, it should have ``feature_importances_`` property
+            (e.g. all tree-based regressors in sklearn).
+            Pre-defined options are also available:
+
+            * catboost: ``catboost.CatBoostRegressor(silent=True)``, this model won't work if there are any category types
```

Comment:
I don't really like that it won't work if any column has type "category".

I tried to somehow dynamically add `cat_features` in `_get_features_weights`, but it can change existing behavior. We should probably discuss how to deal with it.

Reply:
I am not sure but cat_features can be specified in `fit`, so we can add them dynamically just checking the type of the model. Another option is to convert all the categoricals to numeric like we do for linear models.